index.html

```diff
@@ -113,68 +114,109 @@ <h2>I’am a Software Developer</h2>
+                        <li class="myRecentWorkSecondLi">Boostrap</li>
+                        <li class="myRecentWorkSecondLi">Ruby</li>
                     </ul>
+                    <button data-model="2" class="seeProject modal-button">See Project</button>
+
                 </div>
                 <div class="context-projects-3">
                     <span>Data Dashboard Healthcare</span>
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
                         industry's standard</p>
-                    <ul>
+                    <ul class="myRecentWorkThirdLi">
                         <li>Html</li>
                         <li>Boostrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button data-model="3" class="seeProject modal-button">See Project</button>
+
                 </div>
                 <div class="context-projects-3">
                     <span>Data Dashboard Healthcare</span>
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
                         industry's standard</p>
-                    <ul>
+                    <ul class="myRecentWorkThirdLi">
                         <li>Html</li>
                         <li>Boostrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button data-model="3" class="seeProject modal-button">See Project</button>
+
                 </div>
             </div>
 
+        </section>
+        <section class="recent-work-model">
+            <div class="recent-work-time">
+                <h2 id="modal-title">Keeping track of hundreds of components website</h2>
+                <i class="fas fa-times modal2"></i>
+            </div>
+
+
+            <ul id="modal-tag">
+
+            </ul>
+            <div class="recent-work-model-container">
+                <div class="recent-work-img">
+                    <img id="modal-img" src="./img/project1.png" alt="">
+                </div>
+                <div class="recent-work-p">
+                    <p id="modal-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                        Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                        took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of
+                        the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with
+                        the releax map lapora verita.</p>
+
+                    <button>See Live<span class="icon1"></span></button>
+                    <button>See Source<span class="icon2"></span></button>
```

Comment:
- [ ] This should be added to the JavaScript object as it is a requirement for the this project. Please do revisit your project requirements. See image below.

![req](https://user-images.githubusercontent.com/65852621/137128968-8b4344aa-508d-498c-9b21-6b1564908b6d.png)